tekton/src/tasks/gitops/gitops-deprovision-suite-workspace.yml.j2

```diff
@@ -66,7 +66,11 @@ spec:
         export OCP_USERNAME=$(cat $ROSA_CONFIG | yq -r '.data.username')
         export ROSA_CLUSTER_ADMIN_PASSWORD=$(cat $ROSA_CONFIG | yq -r '.data.admin_password')
         oc login -u $OCP_USERNAME -p $ROSA_CLUSTER_ADMIN_PASSWORD $OCP_SERVER --insecure-skip-tls-verify
-
+        OC_RC=$?
+        if [ $OC_RC -ne 0 ]; then
+          echo "oc login failed, exiting with error.."
+          exit 1
+        fi
```

Comment:
I think we have to be careful here. The cluster might not exist but we still want to carry on and remove the pieces from github. This then links to the other piece of work which is to allow deprovsion to run so we need to timeout the argo sync check. So I think we should log the error here but carry on.

Reply:
added the logic to exit with 0 in deprovision flow